.github/workflows/migration-tests.yml

```diff
@@ -21,10 +17,10 @@ jobs:
       config: ${{ steps.filter.outputs.config }}
       advanced_cluster: ${{ steps.filter.outputs.advanced_cluster }}
       backup_online_archive: ${{ steps.filter.outputs.backup_online_archive }}
-      shouldTriggerResourceTest: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || inputs.parent-event-name == 'release' }}
```

Comment:
what was the reson why we had this?

Reply:
the release part is a leftover that was used before in relation with the release process, but not used any more, @andreaangiolillo told me this

Reply:
This is the part that can be removed  `inputs.parent-event-name == 'release'` and it is no longer used